tests/test_orbital.py

```diff
@@ -994,3 +985,39 @@ def orbfit_looks(request):
     x_lk = request.param
     y_lk = np.random.choice([2, 3, 4])
     return x_lk, y_lk
+
+
+class TestOrbfitIndependentMethodWithMultilooking:
+
+    @classmethod
+    def setup_class(cls):
+        cls.conf = TEST_CONF_GAMMA
+        params = Configuration(cls.conf).__dict__
+        conv2tif.main(params)
+        params = Configuration(cls.conf).__dict__
+        prepifg.main(params)
+        cls.params = Configuration(cls.conf).__dict__
+        correct._copy_mlooked(cls.params)
+        correct._create_ifg_dict(cls.params)
+
+    @classmethod
+    def teardown_class(cls):
+        shutil.rmtree(cls.params[C.OUT_DIR])
+
+    def test_independent_method_works_with_multilooking(self, orbfit_looks, orbfit_degrees, orbfit_method=1):
```

Comment:
It looks like this test just checks that the correction actually executes and doesn't change the ifg shape. We should have some check of the value of the correction for a known synthetic ifg.